packages/e2e-tests/specs/editor/blocks/cover.test.js

```diff
@@ -7,6 +15,30 @@ import {
 	openDocumentSettingsSidebar,
 } from '@wordpress/e2e-test-utils';
 
+async function upload( selector ) {
+	await page.waitForSelector( selector );
+	const inputElement = await page.$( selector );
+	const testImagePath = path.join(
+		__dirname,
+		'..',
+		'..',
+		'..',
+		'assets',
+		'1024x768_e2e_test_image_size.jpg'
+	);
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.jpg' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	return filename;
+}
+
+async function waitForImage( filename ) {
+	await page.waitForSelector(
+		`.wp-block-cover img[src$="${ filename }.jpg"]`
+	);
+}
+
```

Comment:
Copied from the Image block e2e test (and found in a few other ones, too -- with slight modifications).

TODO: Extract (into `e2e-utils`?) and reuse.

Reply:
Image upload sounds like a great one to extract 💯

Reply:
I'll add a TODO note in the code for now since I'd otherwise end up touching (and refactoring) a bunch of unrelated files and inflating the PR.